dev-util/vulkan-headers/vulkan-headers-1.2.133.ebuild

```diff
@@ -3,29 +3,26 @@
 
 EAPI=7
 
-inherit cmake-utils
+MY_PN=Vulkan-Headers
+inherit cmake
 
-if [[ "${PV}" == "9999" ]]; then
-	EGIT_REPO_URI="https://github.com/KhronosGroup/Vulkan-Headers.git"
+if [[ ${PV} == *9999* ]]; then
+	EGIT_REPO_URI="https://github.com/KhronosGroup/${MY_PN}.git"
```

Comment:
Have we confirmed upstream uses git tags correctly so we don't need the commit number for a given version?

Reply:
I'm not sure what that is about, but I did notice that vulkan-headers-1.2.134 tar looked somewhat broken, so I did not bump it.

Reply:
Nevermind I can't read

Reply:
> Have we confirmed upstream uses git tags correctly so we don't need the commit number for a given version?

I'm not sure I understand. How can tags be used incorrectly?

Reply:
Sorry, I wanted to make sure we weren't trying to do a git clone of "thing.com/1.2.333.git" or "thing.com/9999.git" but thats not at all what's happening. I read it wrong.

Reply:
Ahh, I see. Okay, thanks for reviewing :)

Reply:
Why fetch with git if we can download the tarball directly? That's what made me miss the update to EGIT_COMMIT. We should not use that in versioned ebuilds that are not live. See https://github.com/KhronosGroup/Vulkan-Headers/releases/tag/v1.2.134 where you can find the links to the tarballs. Let's use that please.